### ISSUE TITLE
Not all python package directories included in zappa zip package

### ISSUE THREAD

**Issue by skruger** [Miserlou/Zappa#1280]
I am trying to deploy my application for the first time and I'm getting import errors when I try to open any URLs.  The top level packages in my application are `config` and `flightclub`.  When I run `zappa package` and inspect the zip file I see that the config directory made it into the package, but the flightclub directory did not.

## Context
To get python 3.6 I created a vagrant instance of Ubuntu 17.10 and initialized it with a virtualenv.  To use it I use `vagrant ssh` to login and then `workon flightclub` to activate the virtualenv.
```
Vagrant.configure(2) do |config|
  config.vm.box = "ubuntu/artful64"
  config.vm.provider "virtualbox" do |vb|
    vb.memory = "1024"
  end
  config.vm.provision "shell", inline: <<-SHELL
    sudo apt-get update
    sudo apt-get install -y build-essential python3-dev python3 python3-virtualenv virtualenv virtualenvwrapper postgresql-9.6 libpq-dev memcached redis-server redis-tools

    su vagrant -c ". /usr/share/virtualenvwrapper/virtualenvwrapper.sh; mkvirtualenv flightclub -p /usr/bin/python3"

    su vagrant -c "/home/vagrant/.virtualenvs/flightclub/bin/pip install -r /vagrant/requirements.txt"
    su vagrant -c "/home/vagrant/.virtualenvs/flightclub/bin/pip install zappa"

    sudo su - postgres -c "createuser flightclub"
    sudo su - postgres -c "createdb flightclub -O flightclub"
    # Base64 encoded alter statement with unescapable quotes sets password to 'flightclub'
    sudo su - postgres -c "echo YWx0ZXIgdXNlciBmbGlnaHRjbHViIHdpdGggcGFzc3dvcmQgJ2ZsaWdodGNsdWIn |base64 -d | psql"

  SHELL
end
```

## Expected Behavior
I expected all python packages in my app directory to be included.  

## Actual Behavior
Only the first python package directory made it into the zip package.

## Your Environment
* Zappa version used: 0.45.1
* Operating System and Python version: Ubuntu 17.10 Python 3.6
* The output of `pip freeze`:
```
argcomplete==1.9.2
base58==0.2.4
boto==2.46.1
boto3==1.4.8
botocore==1.8.6
certifi==2017.11.5
cfn-flip==0.2.5
chardet==3.0.4
click==6.7
decorator==4.1.2
defusedxml==0.5.0
dj-database-url==0.4.2
Django==1.10.6
django-allauth==0.31.0
django-storages-redux==1.3.2
docutils==0.14
durationpy==0.5
freezegun==0.3.9
future==0.16.0
gunicorn==19.7.1
hjson==3.0.1
idna==2.6
ipython==5.3.0
ipython-genutils==0.2.0
jmespath==0.9.3
kappa==0.6.0
lambda-packages==0.19.0
oauthlib==2.0.6
olefile==0.44
pexpect==4.3.0
pickleshare==0.7.4
Pillow==4.0.0
pkg-resources==0.0.0
placebo==0.8.1
prompt-toolkit==1.0.15
psycopg2==2.7.1
ptyprocess==0.5.2
Pygments==2.2.0
python-dateutil==2.6.1
python-slugify==1.2.4
python3-openid==3.1.0
pytz==2017.2
PyYAML==3.12
requests==2.18.4
requests-oauthlib==0.8.0
s3transfer==0.1.12
simplegeneric==0.8.1
six==1.11.0
stripe==1.55.2
toml==0.9.3
tqdm==4.19.1
traitlets==4.3.2
troposphere==2.1.1
Unidecode==0.4.21
urllib3==1.22
wcwidth==0.1.7
Werkzeug==0.12
whitenoise==3.3.0
wsgi-request-logger==0.4.6
zappa==0.45.1
```
* Link to your project (optional): (private)
* `zappa_settings.json`: 
```
{
    "dev": {
        "aws_region": "us-west-2",
        "django_settings": "config.settings.zappa",
        "profile_name": null,
        "project_name": "flightclub",
        "runtime": "python3.6",
        "s3_bucket": "zappa-kqhc7163s",
        "use_precompiled_packages": "false"
    }
}
```

**Comment by skruger** [comment]
I was reading through the source to understand how packages are built and I found a relevant warning in zappa/core.py line 495 that matched my use case, but the exact configuration of my environment didn't cause it to be printed.
```
            print(
                "Warning! Your project and virtualenv have the same name! You may want "
                "to re-create your venv with a new name, or explicitly define a "
                "'project_name', as this may cause errors."
            )
```
I realized that my virtualenv did match the main package directory of my project that was missing.  When I recreated the virtualenv with a new unique name it started working.

**Comment by scoates** [comment]
Seems resolved-enough by #1415. Closing.
